raiden/tests/unit/transfer/mediated_transfer/test_mediatorstate.py

```diff
@@ -1814,6 +1814,102 @@ def test_filter_reachable_routes():
     assert possible_routes[1] not in filtered_routes
 
 
+def test_resume_waiting_transfer():
+    """ Test that a mediator who has a waiting_transfer
+    set (the transfer couldn't be sent forward or backward
+    due to availability or capacity or timeout issues) will retry
+    mediating the waiting_transfer as soon as this transfer's
+    lock expiration becomes valid.
+    """
+    setup = factories.make_transfers_pair(2)
+
+    # Also add transfer sender channel
+    partner_state = factories.NettingChannelEndStateProperties(address=UNIT_TRANSFER_SENDER)
+    payer_channel = factories.create(
+        factories.NettingChannelStateProperties(partner_state=partner_state)
+    )
+    setup.channels.channels.append(payer_channel)
+
+    from_hop = factories.make_hop_from_channel(payer_channel)
+    possible_routes = setup.channels.get_routes()
+
+    # we received a transfer that expires after settle timeout would hit
```

Comment:
```suggestion
    # we received a transfer has an expiry set to a larger value than our settle_timeout. 
    # The `is_channel_usable_for_new_transfer` would return False because of the above. 
    # Therefore, we assign this transfer to `waiting_transfer` and wait until the current block falls
    # into the range `reveal_timeout > lock expiration >= settle_timeout`
```

Reply:
:P just nitpicking here: `expiry` is a (fix) block number, whereas `settle_timeout` is a number of blocks. Therefore the `expiry` value cannot really be compared (`"larger...than"`) to `settle_timeout`.
Nevertheless, thank you for the fix, I believe if it is easier for you to read, so is it to everyone else!